l2geth/core/state/statedb.go

```diff
@@ -244,6 +245,7 @@ func (s *StateDB) GetBalance(addr common.Address) *big.Int {
 	if rcfg.UsingBVM {
 		// Get balance from the bvm_ETH contract.
 		// NOTE: We may remove this feature in a future release.
+		statedumper.WriteETH(addr)
```

Comment:
why we need to `WriteETH` in the geBalance

Reply:
We want to collect addresses which might have non-zero native token balance. It has no impact if we collect some address which do have zero balance. Besides, I found optimism added this code, so I did too.